components/nordigen/nordigen.app.mjs

```diff
@@ -1,11 +1,61 @@
+import axios from "axios";
+
 export default {
   type: "app",
   app: "nordigen",
-  propDefinitions: {},
+  propDefinitions: {
+    country_code: {
+      type: "string",
+      label: "Enter a country code",
+      description: "ISO 3166 two-character code for the country (eg. `FR`). Full list: https://www.iso.org/obp/ui/en/#iso:pub:PUB500001:en",
+    },
+    institution_id: {
+      type: "string",
+      label: "Institution",
+      description: "Select your institution",
+      async options({ country_code }) {
+        const institutions = await this.listInstitutions(country_code); //TODO: Why isn't it working ?
+        return institutions.map((institution) => {
+          return {
+            label: institution.name,
+            value: institution.id,
+          };
+        });
+      },
+    }
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _getHost() {
+      return "https://ob.nordigen.com/api/v2";
+    },
+    _getHeaders() {
+      return {
+        "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
+        "accept": "application/json",
+        "Content-Type": "application/json",
+      }
+    },
+    _getAxiosParams(opts) {
+      return {
+        ...opts,
+        url: this._getHost() + opts.path,
+        headers: this._getHeaders()
+      }
+    },
+    async _makeRequest(method, endpoint, data, params) {
+	  const config = {
+	  };
+      return await axios({	//TODO: use platform.axios instead
```

Comment:
No need to use `platform.axios` here. Its not possible to use `pipedream.axios` in the `app`. So its fine to use the normal `Axios` here. Try to use the `platform.axios` on the sources and actions :)

Reply:
No need to `await` here as well.
An  `async` function will automatically `await` a returned promise